src/sentry/sentry_metrics/querying/metadata/metrics_code_locations.py

```diff
@@ -161,6 +165,8 @@ def fetch(self) -> Sequence[MetricCodeLocations]:
             # de-duplication.
             code_locations += self._get_code_locations(queries)
 
+        metrics.distribution("ddm.metrics_code_locations.fetched", value=len(code_locations))
```

Comment:
This is a very lossy number but it would help to give us an idea of how many code locations are fetched across time.